x/bandoracle/types/errors.go

```diff
@@ -8,6 +8,7 @@ import (
 var ErrorUnknownProposalType = errors.Register(ModuleName, 401, "unknown proposal type")
 
 var (
-	ErrSample         = errors.Register(ModuleName, 1100, "sample error")
-	ErrInvalidVersion = errors.Register(ModuleName, 1501, "invalid version")
+	ErrSample             = errors.Register(ModuleName, 1100, "sample error")
```

Comment:
@pratikasr please replace this error with relevant message wherever it is being used

Reply:
Done.